module_1/src/App.js

```diff
@@ -0,0 +1,25 @@
+
+import './App.css';
+import Counter from "./components/Counter.js";
+import SearchForm from './components/SearchForm.js';
+import GenreSelect from './components/GenreSelect.js';
+
+function App() {
+  var searchFormProps = {};
+  searchFormProps.initialQuery = "horror";
+  searchFormProps.onSearch = (query) => {console.log("search initiated with query: " + query)}
+
+  var genreSelectProps = {};
+  genreSelectProps.genreList = ["horror","comedy","fantasy","docu"];
+  genreSelectProps.selected = "fantasy";
+  genreSelectProps.onSelect = (genre) => {console.log("genre selected: " + genre)}
+  return <>
+          <Counter /> 
+          <SearchForm { ...searchFormProps} />
+          <GenreSelect { ...genreSelectProps} />
```

Comment:
If you are in an experimenting mood, you can directly provide the props, no need to create a variable for it:
```
<GenreSelect
    genreList={["horror","comedy","fantasy","docu"]}
    selected="fantasy"
    onSelect={(genre) => {console.log("genre selected: " + genre)}}
```